### PR TITLE
iasRtdProvider: implements getTargetingData method

### PR DESCRIPTION
**Pull request by raguilar-ias**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by ChrisHuie** [comment]
errors here are related to an ie issue in another pr that was just reverted. Pulling in recent merges again should fix errors here

**Comment by raguilar-ias** [comment]
> errors here are related to an ie issue in another pr that was just reverted. Pulling in recent merges again should fix errors here

Errors fixed.

File: modules/iasRtdProvider.js [hunk 1]
```diff
@@ -29,10 +31,18 @@ function stringifySlotSizes(sizes) {
...
-function stringifySlot(bidRequest) {
+function stringifySlot(bidRequest, adUnitPath) {
+  var sizes = [];
+  if ('mediaTypes' in bidRequest) {
+    for (const type in bidRequest.mediaTypes) {
+      if (bidRequest.mediaTypes.hasOwnProperty(type) && type == 'banner') {
+        sizes.push(bidRequest.mediaTypes[type].sizes)
+      }
+    }
+  }
```
Comment (ChrisHuie) [inline modules/iasRtdProvider.js:42]: Would the built in utils function for getAdUnitSizes help here? https://github.com/prebid/Prebid.js/blob/b4cf8b11e57e4df6b12582de68690d7fc1a25e8f/src/utils.js#L148

Comment (raguilar-ias) [reply modules/iasRtdProvider.js]: @ChrisHuie I updated with the function getAdUnitSizes you suggested.

**Pull request merged**